config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -685,3 +685,36 @@ periodics:
       env:
       - name: GOPATH
         value: /go
+
+- name: ci-benchmark-kube-dns-master
+  interval: 2h
+  tags:
+  - "perfDashPrefix: kube-dns benchmark"
+  - "perfDashJobType: dnsBenchmark"
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20181205-915278e90-master
+      args:
+      - --repo=k8s.io/kubernetes=master
+      - --repo=k8s.io/perf-tests=master
+      - --root=/go/src
+      - --timeout=140
+      - --scenario=kubernetes_e2e
+      - --
+      - --check-leaked-resources
+      - --cluster=kube-dns-benchmark
+      - --extract=ci/latest
+      - --gcp-nodes=3
+      - --gcp-project-type=scalability-project
```

Comment:
We don't need scalability project, if the cluster is supposed to have 3 nodes.
Scalability projects have 100-node quota, so we don't want to waste them if not needed.

Reply:
I removed `scalability-project` requirement. I assume boskos will find suitable project. Or is it better to assign specific project?